html/template/default/assets/scss/style.scss

```
@@ -70,3 +70,4 @@ p {
 @import "project/20.1.404";
 @import "project/21.1.withdraw";
 @import "project/22.1.editComplete";
+@import "project/23.1.faq";
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)